lib/Conversion/TorchToLinalg/TorchToLinalg.cpp

```diff
@@ -2783,9 +2783,14 @@ class ConvertAtenOnesOp : public OpConversionPattern<AtenOnesOp> {
     if (!adaptor.layout().getType().isa<Torch::NoneType>())
       return rewriter.notifyMatchFailure(op,
                                          "only default layout is supported");
+
     bool pinMemory;
```

Comment:
pinMemory should be initialized to true here. In case `pin_memory` is Torch::NoneType, pinMemory can end up with random init value.

Reply:
I think the default value is `False`, but yes I will initialize it.

This is how Pytorch converts it

https://github.com/pytorch/pytorch/blob/dc24503a89a1aefaab73e44124d4fa563f4616d1/c10/core/TensorOptions.h#L44-L46

Reply:
ohh, I meant the `bool pinMemory` in the c++ code could be initialized with any value including true. And the `if (pinMemory)` check could report an error when `pin_memory` is None.